stubs/openpyxl/openpyxl/worksheet/worksheet.pyi

```diff
@@ -24,106 +31,106 @@ class Worksheet(_WorkbookChild):
+    def add_table(self, table: Table) -> None: ...
     @property
-    def tables(self): ...
-    def add_pivot(self, pivot) -> None: ...
+    def tables(self) -> TableList: ...
+    def add_pivot(self, pivot: Any) -> None: ...
     def merge_cells(
         self,
-        range_string: Incomplete | None = ...,
-        start_row: Incomplete | None = ...,
-        start_column: Incomplete | None = ...,
-        end_row: Incomplete | None = ...,
-        end_column: Incomplete | None = ...,
+        range_string: str | None = ...,
+        start_row: int | None = ...,
+        start_column: int | None = ...,
+        end_row: int | None = ...,
+        end_column: int | None = ...,
     ) -> None: ...
     @property
-    def merged_cell_ranges(self): ...
+    def merged_cell_ranges(self) -> list[CellRange]: ...
     def unmerge_cells(
         self,
-        range_string: Incomplete | None = ...,
-        start_row: Incomplete | None = ...,
-        start_column: Incomplete | None = ...,
-        end_row: Incomplete | None = ...,
-        end_column: Incomplete | None = ...,
+        range_string: str | None = ...,
+        start_row: int | None = ...,
+        start_column: int | None = ...,
+        end_row: int | None = ...,
+        end_column: int | None = ...,
     ) -> None: ...
-    def append(self, iterable) -> None: ...
-    def insert_rows(self, idx, amount: int = ...) -> None: ...
-    def insert_cols(self, idx, amount: int = ...) -> None: ...
-    def delete_rows(self, idx, amount: int = ...) -> None: ...
-    def delete_cols(self, idx, amount: int = ...) -> None: ...
-    def move_range(self, cell_range, rows: int = ..., cols: int = ..., translate: bool = ...) -> None: ...
-    @property
-    def print_title_rows(self): ...
+    def append(self, iterable: Iterable[Any]) -> None: ...
+    def insert_rows(self, idx: int, amount: int = ...) -> None: ...
+    def insert_cols(self, idx: int, amount: int = ...) -> None: ...
+    def delete_rows(self, idx: int, amount: int = ...) -> None: ...
+    def delete_cols(self, idx: int, amount: int = ...) -> None: ...
+    def move_range(self, cell_range: CellRange, rows: int = ..., cols: int = ..., translate: bool = ...) -> None: ...
+    @property
+    def print_title_rows(self) -> Any: ...
     @print_title_rows.setter
-    def print_title_rows(self, rows) -> None: ...
+    def print_title_rows(self, rows: int) -> None: ...
     @property
-    def print_title_cols(self): ...
+    def print_title_cols(self) -> Any: ...
     @print_title_cols.setter
-    def print_title_cols(self, cols) -> None: ...
+    def print_title_cols(self, cols: int) -> None: ...
     @property
-    def print_titles(self): ...
+    def print_titles(self) -> Any: ...
     @property
-    def print_area(self): ...
+    def print_area(self) -> Any: ...
     @print_area.setter
-    def print_area(self, value) -> None: ...
+    def print_area(self, value: str) -> None: ...
```

Comment:
This is incorrect, it also accepts an iterable of strings: https://foss.heptapod.net/openpyxl/openpyxl/-/blob/branch/3.0/openpyxl/worksheet/worksheet.py#L883-891